### PR TITLE
Allow selection of unstable env - fix setup

### PR DESCRIPTION
**Pull request by ezdac**
This PR is fixing an issue there was with #951.
It will include the commit of the fixed `raiden-contracts` version (https://github.com/raiden-network/raiden-contracts/pull/1448) and will not use the `--no-use-pep517` pip flag.
(see https://github.com/raiden-network/raiden-services/pull/951#issuecomment-814959179)

### CODE REVIEW

**Comment by ezdac** [comment]
To me this looks good - but I would wait until the contracts are merged, since there are open questions and the commit hash is used in the requirements here.

**Review by karlb — APPROVED** [review]

**Pull request merged**
